doc/migration/v0.1.0-migration-guide.md

```diff
@@ -299,7 +299,7 @@ At this point you should be able to build and run your operator to verify that i
 [v0.1.0-changes-doc]: ./v0.1.0-changes.md
 [v0.0.7-memcached-operator]: https://github.com/operator-framework/operator-sdk-samples/tree/aa15bd278eec0959595e0a0a7282a26055d7f9d6/memcached-operator
 [v0.1.0-memcached-operator]: https://github.com/operator-framework/operator-sdk-samples/tree/4c6934448684a6953ece4d3d9f3f77494b1c125e/memcached-operator
-[controller-conventions]: https://github.com/kubernetes/community/blob/master/contributors/devel/controllers.md#guidelines
+[controller-conventions]: https://github.com/kubernetes/community/blob/cbe9c8ac5f71a99179d7ffe4a008b9018830af72/contributors/devel/sig-api-machinery/controllers.md#guidelines
```

Comment:
Should we still use `master` so users see the updates? I realize there's a tradeoff, which is that we have to have these PRs to fix broken links every now and then, but wondering if we think its worth it?

Reply:
Yes, I am divided on this, either way makes sense to me, up to the team.

Reply:
I believe that we should use the permalink, and have a reoccurring task that the team needs to update it if it changed. 

I think this because I don't want to block the user's PR's with the marker test for something that they can't control.

Reply:
I have an issue open for this already https://github.com/crawford/marker/issues/18 My rust is rather rusty (sorry) also not sure if the maintainer accepts new features.

Reply:
I agree, permalinking cements the details we cared about at the time and prevents user confusion.